packages/connector-tencent-sms/src/index.test.ts

```diff
@@ -0,0 +1,73 @@
+import { MessageTypes } from '@logto/connector-kit';
+
+import createConnector from '.';
+import { sendSMSRequest } from './http';
+import { codeTest, mockedConnectorConfig, mockedTemplateCode, phoneTest } from './mock';
+
+const getConfig = jest.fn().mockResolvedValue(mockedConnectorConfig);
+
+jest.mock('./http', () => {
+  return {
+    sendSMSRequest: jest.fn(() => {
+      return {
+        body: {
+          Response: {
+            RequestId: 'request-id',
+            SendStatusSet: [
+              {
+                Fee: 1,
+                SerialNo: 'serial-no',
+                SessionContext: 'session-context',
+                Code: 'Ok',
+                Message: 'OK',
```

Comment:
nit: Can we unify the letter case?
```suggestion
                Code: 'OK',
                Message: 'OK',
```

Reply:
Tencent Cloud returns `Ok` not `OK`

Reply:
And their message is `OK` in full uppercase?